services/cards-publication/src/test/java/org/opfab/cards/publication/services/CardProcessServiceShould.java

```diff
@@ -412,6 +421,71 @@ void GIVEN_an_invalid_card_WHEN_sending_card_THEN_card_is_rejected() {
                 Assertions.assertThat(checkArchiveCount(0)).isTrue();
         }
 
+        @Test
+        void GIVEN_a_parent_card_WHEN_sending_a_child_card_THEN_card_has_startDate_and_endDate_correctly_set() throws URISyntaxException {
+
+                Card card = generateOneCard();
+                cardProcessingService.processCard(card);
+
+                ArrayList<String> externalRecipients = new ArrayList<>();
+                externalRecipients.add(API_TEST_EXTERNAL_RECIPIENT_1);
+
+                Card childCard = Card.builder().publisher("newPublisherId")
+                                .processVersion("0")
+                                .processInstanceId("PROCESS_CARD_USER").severity(SeverityEnum.INFORMATION)
+                                .process("PROCESS_CARD_USER")
+                                .parentCardId(card.getId())
+                                .initialParentCardUid(card.getUid())
+                                .title(new I18n("title",null))
+                                .summary(new I18n("summary",null))
+                                .startDate(Instant.now())
+                                .state("state1")
+                                .build();
+
+                Assertions.assertThatCode(
+                                () -> cardProcessingService.processUserCard(childCard, currentUserWithPerimeters,
+                                                token))
+                                .doesNotThrowAnyException();
+                Assertions.assertThat(checkChildCardsDates(childCard, card)).isTrue();
+        }
```

Comment:
instead of using method checkChildCardsDates , i suggest having a precise assert 

Like : 

```
assertThat(chidcard.startDate).equals(card.publishDate)
assertThat(chidcard.endDate).equals(card.publishDate)
```

The idea is to ease the understanding of the test